src/functions/get/domain/config.mock.ts

```diff
@@ -0,0 +1,9 @@
+export const config = {
+  GoogleAnalyticsID: 'dev-ga-id',
```

Comment:
What's the reason for InitCaps in here vs camelCase in the app?

Reply:
Have changed to camelCase, thanks for spotting :)